packages/react-sdk/src/core-components.ts

```diff
@@ -1 +1,3 @@
+export const portalComponent = "Slot";
```

Comment:
I am fine with renaming it btw to Portal, whenever the time is right

Reply:
Would be more like ws:portal